cli/popper/commands/cmd_run.py

```diff
@@ -223,6 +223,8 @@ def run_workflow(**kwargs):
             pre_wf_runner = WorkflowRunner(pre_wf)
             pre_wf_runner.run(**kwargs)
 
+        for key, value in kwargs.items(): 
+            print ("%s == %s" %(key, value))
```

Comment:
@ashadhaz, this is causing the tests to fail. If you remove this the tests should pass. thanks!